ansible/roles/apache/templates/apache2.conf.j2

```diff
@@ -0,0 +1,56 @@
+#ServerRoot "/etc/apache2"
+Mutex file:${APACHE_LOCK_DIR} default
+PidFile ${APACHE_PID_FILE}
+Timeout 300
+KeepAlive On
+MaxKeepAliveRequests 100
+KeepAliveTimeout 5
+User vagrant
+Group vagrant
+HostnameLookups Off
+ErrorLog ${APACHE_LOG_DIR}/error.log
+LogLevel warn
+
+# Include module configuration:
+IncludeOptional mods-enabled/*.load
```

Comment:
Do we need any of this includes? Failure while provisioning gets probably bigger with each of them. Except the application.conf. a2enmod a2dismod breaks the behaviour and we have no idea to support problems. That's why the nginx config is that small. I am sure this runs well as long a user does not break it with no idea what he's done